### PR TITLE
Fix wrong path and missing HTTP Client in Windows scripts

### PR DESCRIPTION
**Pull request by romainmoreau**
Please see the issue : https://github.com/moai/moai-dev/issues/881
This fix also activates OpenSSL for HTTPS and fix a path issue.

### CODE REVIEW

File: bin/build-win.bat [hunk 1]
```diff
@@ -1,6 +1,6 @@
-cd cmake
+cd ..\cmake
```
Comment (halfnelson) [inline bin/build-win.bat:1]: This wasn't wrong path. Docs say to run this script from root of sdk like: bin\build-win.bat which needs cd cmake

Comment (romainmoreau) [inline bin/build-win.bat:1]: You are absolutely right, I reverted the change and submitted another pull request.
